junitparser/junitparser.py

```diff
@@ -470,6 +470,12 @@ def add_testcase(self, testcase):
         self.append(testcase)
         self.update_statistics()
 
+    def add_testcases(self, testcases):
+        """Adds a testcases to the suite."""
+        for testcase in testcases:
```

Comment:
The loop could be made redundant if there would be a `def extend` that calls into `self._elem.extend`, which is the equivalent of `append` for a list.

Reply:
It cab be done only that way
``` python
def extend(self, sub_elems):
        self._elem.extend((sub_elem._elem for sub_elem in sub_elems))
```
Is it ok?

Reply:
done